content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -0,0 +1,308 @@
+  verbs:
+  - update
+- apiGroups:
+  - certificates.k8s.io
+  resources:
+  - signers
+  resourceName:
+  - example.com/my-signer-name # example.com/* can be used to authorize for all signers in the 'example.com' domain
+  verbs:
+  - sign
+```
+
+## Approval/Rejection
+
+### kube-controller-manager auto-approval
+
+The kube-controller-manager ships with a built-in approver for certificates with
+a signerName of `kubernetes.io/kube-apiserver-client-kubelet` that delegates various
+permissions on CSRs for node credentials to authorization.
+It does this by posting SubjectAccessReview resources to the API server.
+
+### kubectl-based approver
+
+A Kubernetes administrator (with appropriate permissions) can manually approve
+(or deny) Certificate Signing Requests by using the `kubectl certificate
+approve` and `kubectl certificate deny` commands.
+
+To approve a CSR with kubectl:
+
+```bash
+kubectl certificate approve <certificate-signing-request-name>
+```
+
+Likewise, to deny a CSR:
+
+```bash
+kubectl certificate deny <certificate-signing-request-name>
+```
+
+### API-based approver
+
+Users of the REST API can approve CSRs by submitting an UPDATE request to the `approval`
+subresource of the CSR to be approved.
+
+As part of this request, the `Approved` or `Denied` status condition should be set to the
+desired state:
+
+For `Approved` CSRs:
+
+```yaml
+apiVersion: certificates.k8s.io/v1beta1
+kind: CertificateSigningRequest
+...
+status:
+  conditions:
+  - lastUpdateTime: "2020-02-08T11:37:35Z"
+    message: Auto approving kubelet client certificate after SubjectAccessReview.
+    reason: AutoApproved # this can be set to anything
+    type: Approved
+```
+
+For `Denied` CSRs:
+
+```yaml
+apiVersion: certificates.k8s.io/v1beta1
+kind: CertificateSigningRequest
+...
+status:
+  conditions:
+  - lastUpdateTime: "2020-02-08T11:37:35Z"
+    message: Denied due to request not matching requirements
+    reason: NotAllowed # this can be set to anything
+    type: Denied
+```
+
+## Signing
+
+### kube-controller-manager signer
+
+The kube-controller-manager ships with a built-in signer that signs certificates
+that have the kubernetes.io signerNames defined above (kube-controller-manager
+versions prior to the introduction of signerName in v1.18 would sign all approved CSRs).
+
+### API-based signers
+
+Users of the REST API can sign CSRs by submitting an UPDATE request to the `status`
+subresource of the CSR to be signed.
+
+As part of this request, the `status.certificate` field should be set to contain the
+signed certificate.
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* kube-controller-manager built in [signer](https://github.com/kubernetes/kubernetes/blob/32ec6c212ec9415f604ffc1f4c1f29b782968ff1/pkg/controller/certificates/signer/cfssl_signer.go)
+* kube-controller-manager built in [approver](https://github.com/kubernetes/kubernetes/blob/32ec6c212ec9415f604ffc1f4c1f29b782968ff1/pkg/controller/certificates/approver/sarapprove.go)
```

Comment:
Consider also signposting readers to go and read https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/ at this point.